canonical/jaas-dashboard
# WD-1230 - Split app utils and selectors

## Done

Move app utils and selectors to appropriate dirs.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that the app functions as before.

## Details

https://warthogs.atlassian.net/browse/WD-1230
